src/builder/actions/Actions.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: BSD-3-Clause
 pragma solidity ^0.8.27;
 
+import {console} from "../console.sol";
```

Comment:
Can't you just import it from Foundry directly? via `import {console} from "forge-std/console.sol"`

Reply:
Yes, I could, but then I get a stack too deep error. Go figure :(